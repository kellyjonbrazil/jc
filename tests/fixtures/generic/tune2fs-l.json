{"version":"1.46.2 (28-Feb-2021)","filesystem_volume_name":"<none>","last_mounted_on":"/home","filesystem_uuid":"5fb78e1a-b214-44e2-a309-8e35116d8dd6","filesystem_magic_number":"0xEF53","filesystem_revision_number":"1 (dynamic)","filesystem_features":["has_journal","ext_attr","resize_inode","dir_index","filetype","needs_recovery","extent","64bit","flex_bg","sparse_super","large_file","huge_file","dir_nlink","extra_isize","metadata_csum"],"filesystem_flags":"signed_directory_hash","default_mount_options":"user_xattr acl","filesystem_state":"clean","errors_behavior":"Continue","filesystem_os_type":"Linux","inode_count":3932160,"block_count":15728640,"reserved_block_count":786432,"free_blocks":15198453,"free_inodes":3864620,"first_block":0,"block_size":4096,"fragment_size":4096,"group_descriptor_size":64,"reserved_gdt_blocks":1024,"blocks_per_group":32768,"fragments_per_group":32768,"inodes_per_group":8192,"inode_blocks_per_group":512,"flex_block_group_size":16,"filesystem_created":"Mon Apr  6 15:10:37 2020","last_mount_time":"Mon Sep 19 15:16:20 2022","last_write_time":"Mon Sep 19 15:16:20 2022","mount_count":14,"maximum_mount_count":-1,"last_checked":"Fri Apr  8 15:24:22 2022","check_interval":"0 (<none>)","lifetime_writes":"203 GB","reserved_blocks_uid":"0 (user root)","reserved_blocks_gid":"0 (group root)","first_inode":11,"inode_size":256,"required_extra_isize":32,"desired_extra_isize":32,"journal_inode":8,"default_directory_hash":"half_md4","directory_hash_seed":"67d5358d-723d-4ce3-b3c0-30ddb433ad9e","journal_backup":"inode blocks","checksum_type":"crc32c","checksum":"0x7809afff","filesystem_created_epoch":1586211037,"filesystem_created_epoch_utc":null,"last_mount_time_epoch":1663625780,"last_mount_time_epoch_utc":null,"last_write_time_epoch":1663625780,"last_write_time_epoch_utc":null,"last_checked_epoch":1649456662,"last_checked_epoch_utc":null}
