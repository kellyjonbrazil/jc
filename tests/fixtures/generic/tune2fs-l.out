tune2fs 1.46.2 (28-Feb-2021)
Filesystem volume name:   <none>
Last mounted on:          /home
Filesystem UUID:          5fb78e1a-b214-44e2-a309-8e35116d8dd6
Filesystem magic number:  0xEF53
Filesystem revision #:    1 (dynamic)
Filesystem features:      has_journal ext_attr resize_inode dir_index filetype needs_recovery extent 64bit flex_bg sparse_super large_file huge_file dir_nlink extra_isize metadata_csum
Filesystem flags:         signed_directory_hash
Default mount options:    user_xattr acl
Filesystem state:         clean
Errors behavior:          Continue
Filesystem OS type:       Linux
Inode count:              3932160
Block count:              15728640
Reserved block count:     786432
Free blocks:              15198453
Free inodes:              3864620
First block:              0
Block size:               4096
Fragment size:            4096
Group descriptor size:    64
Reserved GDT blocks:      1024
Blocks per group:         32768
Fragments per group:      32768
Inodes per group:         8192
Inode blocks per group:   512
Flex block group size:    16
Filesystem created:       Mon Apr  6 15:10:37 2020
Last mount time:          Mon Sep 19 15:16:20 2022
Last write time:          Mon Sep 19 15:16:20 2022
Mount count:              14
Maximum mount count:      -1
Last checked:             Fri Apr  8 15:24:22 2022
Check interval:           0 (<none>)
Lifetime writes:          203 GB
Reserved blocks uid:      0 (user root)
Reserved blocks gid:      0 (group root)
First inode:              11
Inode size:               256
Required extra isize:     32
Desired extra isize:      32
Journal inode:            8
Default directory hash:   half_md4
Directory Hash Seed:      67d5358d-723d-4ce3-b3c0-30ddb433ad9e
Journal backup:           inode blocks
Checksum type:            crc32c
Checksum:                 0x7809afff

